### ISSUE TITLE
[Brazilian Portuguese] Introduction to Web Accessibility

### ISSUE THREAD

**Issue by adielcristo** [w3c/wai-translations#253]
### Page URL

https://www.w3.org/WAI/fundamentals/accessibility-intro/

### Instructions

- [X] I have read the information on [Translating WAI Documents](https://www.w3.org/WAI/about/translating/)
- [X] Before starting the translation, I will wait for confirmation that the resource is ready for translation.

### Link to original file in GitHub

https://github.com/w3c/wai-website/blob/main/pages/fundamentals/introduction/index.md

### PRs

Page: https://github.com/w3c/wai-website/pull/924
Menu: https://github.com/w3c/wai-website-data/pull/176

**Comment by remibetin** [comment]
Hi @adielcristo,

The title of your pull request indicates your translation is ready for review. Can you confirm this?

Thanks!

**Comment by adielcristo** [comment]
Hi @remibetin,

Yes, it is ready for review. I also sent a PR for the menu title: https://github.com/w3c/wai-website-data/pull/176.

**Comment by remibetin** [comment]
This translation is ready for review! 🎉

Reviewers are welcome to add their comments and collaborate with the translator in the related Pull Request: https://github.com/w3c/wai-website/pull/924

Guidance: https://www.w3.org/WAI/about/translating/guides/review-translation/

**Comment by israelcefrin** [comment]
Hello @adielcristo and @remibetin, I will review this translation. It will be complete by December, 9th 2024.

**Comment by israelcefrin** [comment]
Hello @adielcristo 

I have completed my review and I am pleased to say that there are only a few items I would like to address. Most of these relate more to formal rules than to the translation itself.
You will find my review below, along with my suggestions. Please feel free to review them and adjust the text if you agree with my comments.

## Alert box:
1) Missing accentuation: word "ingles" should be "inglês"

## Related resource/recurso relacionado:
1) Yet the video has subtitles, it would be helpful if letting the users know that the audio is in English. It may be achieved including this information in the link label. E.g.: Vídeo Introdução à acessibilidade web e Padrões W3C (4 minutos, vídeo em inglês).

## Accessibilidade em Contexto
1) The term "*Web*" is usually written with capital "W". In Brazilian Portuguese we also noted it as in English. E.g. https://www.w3.org/Translations/WCAG20-pt-br/ . I would recommend an extended replace to keep the uniformity with other W3C documents throughout the whole document.

2) Adjusting the verbal mode from "indicativo" to "subjuntivo" (a wishfull thinking, not yet a reality but it is possible to become true): 
 pessoas desenvolvedoras e organizações que _desejam_ => pessoas desenvolvedoras e organizações que _desejem_


## O que é acessibilidade web
1)  Noun "óculos" is plural and requires that adjectives or verbs flex with it:  um braço quebrado ou óculos _perdido_ => um braço quebrado ou óculos _perdidos_ . Another option would be changing it to: um braço quebrado ou _um par_ de óculos perdido. 

2) Vídeo Perspectivas da acessibilidade web (YouTube) => Vídeo Perspectivas da acessibilidade web (YouTube, _vídeo em inglês_)

### Mais informações sobre o que é acessibilidade
1) The noun "_inglês_"(English) doesn't require initial capital letter in Brazilian Portuguese. Also, the official WCAG Brazilian Portuguese translation uses "inglês": https://www.w3.org/Translations/WCAG20-pt-br/
Recommended to replace every "em Inglês" by "em inglês" 

## Acessibilidade é importante para pessoas, negócios e a sociedade
1) The link to CRPD should include a label to let users know that the link target content is in English: 
sobre os Direitos das Pessoas com Deficiência (UN CRPD _texto em inglês_).

2) "Mídia impressa" is connected mostly to newspaper and it doesn't relate to other print material like journals or books. I think that we should change it to include any print media. Also "visual media" is a single term to any form of communication or expression that relies on visual elements, e.g.: television, photography, Art, etc... The noun "media" in this sentence relates only to "visual". Hence, I suggest to reword the statement from:
as barreiras de acessibilidade à mídia impressa, de áudio e visual
To:
as barreiras de acessibilidade _a materiais impressos, áudio e mídia visual_...

Also, removing the "crase" from the letter "a" because it relates to more than one element(plural) and it becomes a preposition instead of "preposition + article"

3) The structure fro the following sentence:
"o design acessível melhora a experiência geral da pessoa usuária e a satisfação"

Could be improved if we keep both criteria together (experiência e satisfação)
"o design acessível melhora a experiência geral e a satisfação da pessoa usuária .."

4) Missing "em inglês" warning in the link label of " Fatores legais e de política.". It should be:
...  Fatores legais e de política(em inglês).  ". 


### Tornando seu site acessível
- Como atender às WCAG (Referência rápida) => Como atender às WCAG (Referência rápida, em inglês)

## Exemplos
###Texto alternativo para imagens
1) Missing the info about link target content in English:
	-  As imagens devem incluir texto alternativo equivalente => As imagens devem incluir texto alternativo equivalente (_em inglês_)

2) The example image is still showing an alt text in English. Will it be image translated?

### Entrada de teclado
1) Missing the info about link target content in English:
	- ele disponibiliza todas as funcionalidades a partir de um teclado => ele disponibiliza todas as funcionalidades a partir de um teclado (em inglês).

### Transcrições para áudio
1) Missing the info about link target content in English:
	- Também existem serviços de transcrição que => Também existem serviços de transcrição(em inglês) que

**Comment by remibetin** [comment]
This translation has been published today 🚀 Thank you @adielcristo @israelcefrin!

See https://www.w3.org/WAI/fundamentals/accessibility-intro/pt-BR

**Comment by israelcefrin** [comment]
Thanks for the update, @remibetin, and thank you, @adielcristo, for all your work translating the content to Brazilian Portuguese. I'm looking forward to working on the translation of other pages.

**Comment by adielcristo** [comment]
Thanks @remibetin, @israelcefrin, for all the help.
